src/Chapter2.hs

```diff
@@ -841,7 +858,9 @@ list.
 
 🕯 HINT: Use the 'cycle' function
 -}
-rotate = error "rotate: Not implemented!"
+rotate :: Int -> [Int] -> [Int]
+rotate n l =
+    take (length l) $ drop n $ cycle l
```

Comment:
Function `rotate` should return empty list on negative numbers, but your implementation will return the list itself